spec/lib/myfinance/resources/payable_account_spec.rb

```diff
@@ -82,4 +84,30 @@
       end
     end
   end
+
+  describe "#pay", vcr: true do
+    let(:params) { { total_amount: 150.99, occurred_at: '2015-08-05', amount: 150.99 } }
+    subject { client.payable_accounts.pay(1215631, entity_id, params) }
+
+    context "with success" do
+      it "pays payable account" do
+        expect(subject.id).to eq(1215631)
+        expect(subject.status).to eq(2)
+        expect(subject.status_name).to eq('paid')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.